using_images/xpaas_images/jws.adoc

```diff
@@ -9,7 +9,66 @@
 
 toc::[]
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+== Overview
+
+The Apache Tomcat 7 and Apache Tomcat 8 components of Red Hat JBoss Web Server 3 are available as containerized xPaaS images that are designed for use with OpenShift. Developers can use these images to quickly build, scale, and test Java web applications deployed across hybrid environments.
+
+[IMPORTANT]
+There are significant differences in the functionality in the JBoss Web Server xPaaS images compared to the regular release of JBoss Web Server.
```

Comment:
Not too sure if "functionality in" is the right preposition. 

Suggestion: "There are significant differences in the functionality between the JBoss Web Server xPaaS images and the regular release of JBoss Web Server." Also is "images" right? Are there more than one?